core/astro-iconify-svgmap/skills/astro-iconify-svgmap/SKILL.md

```
@@ -0,0 +1,216 @@
+
+Generates SVG sprite maps from Iconify icon packs at build time and serves them during dev. Icons are registered lazily via `getIcon()` and assembled into per-pack sprite files.
+
+## Setup
+
+1. Install the integration and at least one `@iconify-json/*` pack:
+
+```sh
+pnpm add @stephansama/astro-iconify-svgmap @iconify-json/mdi
+```
+
+2. Add to `astro.config.mjs`:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)